src/main/scala/lenthall/numeric/IntegerUtil.scala

```diff
@@ -0,0 +1,18 @@
+package lenthall.numeric
+
+object IntegerUtil {
+
+  private val OrdinalSuffixes = List("th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th")
```

Comment:
Would this replace only the list or the entire `toOrdinal` method ?
If the former I think `int +` shouldn't be there and if the latter it wouldn't work for 21st, 22nd etc..